lib/pages/laws.dart

```diff
@@ -5,47 +5,169 @@ import 'package:heroes_manual/utility/colors.dart';
 
 class Laws extends StatelessWidget {
   static const String route = '/laws';
+  static const List<String> GA_laws = [
```

Comment:
move this content to utility/strings.dart and reference it in the source code below